### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Fred-Leclercq**
Typo.

### CODE REVIEW

**Comment by m-mohr** [comment]
Thank you!

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ You can use any file format, but here is a list of common media types for foreca
 | Type                    | Description |
 | ----------------------- | ----------- |
 | `application/wmo-GRIB2` | GRIB2 file (usually with the file extension `.grb2` or `.grib2`) |
-| `application/netcdf`    | NetCDF file (usually with the file extension `.nc`) |
+| `application/x-netcdf`    | NetCDF file (usually with the file extension `.nc`) |
```
Comment (m-mohr) [inline README.md]: I've seen both being used. Do you have a source about which is recommended to use by the authors of the spec or so?

Comment (Fred-Leclercq) [reply README.md]: I could not find it in IANA: https://www.iana.org/assignments/media-types/media-types.xhtml

"The x- prefix of a MIME subtype-identifier implies that it is non-standard i.e. not registered with IANA" - https://www.nih-cfde.org/resource/intro-mime-type/

Comment (m-mohr) [reply README.md]:
```suggestion
| `application/x-netcdf`  | NetCDF file (usually with the file extension `.nc`) |
```

**Pull request merged**